test/Conversion/amd/tritongpu_wmma_dot_to_llvm.mlir

```diff
@@ -4,6 +4,7 @@
 #shared = #ttg.shared<{vec = 1, perPhase = 1, maxPhase = 1, order = [1, 0], hasLeadingOffset = false}>
 #mma1 = #ttg.amd_wmma<{version = 1, warpsPerCTA = [2, 2]}>
 #mma2 = #ttg.amd_wmma<{version = 2, warpsPerCTA = [2, 2]}>
+#mma3 = #ttg.amd_wmma<{version = 2, warpsPerCTA = [2, 2], isTranspose = true}>
```

Comment:
could you please rename the layout to ``mma_transposed`` or something?
``mma1/2`` means version 1/2. If you want, feel free to rename to ``mma_ver1/2``.